hydra-cluster/test/Test/EndToEndSpec.hs

```diff
@@ -313,11 +335,62 @@ spec = around showLogsOnFailure $ do
                     metrics <- getMetrics n1
                     metrics `shouldSatisfy` ("hydra_head_events" `BS.isInfixOf`)
 
-    describe "hydra-node executable" $
+    describe "hydra-node executable" $ do
       it "display proper semantic version given it is passed --version argument" $ \_ ->
         failAfter 5 $ do
           version <- readCreateProcess (proc "hydra-node" ["--version"]) ""
           version `shouldSatisfy` (=~ ("[0-9]+\\.[0-9]+\\.[0-9]+(-[a-zA-Z0-9]+)?" :: String))
+      fit "hydra-node logs it's command line arguments" $ \tracer -> do
+        failAfter 60 $
+          withTempDir "temp-dir-to-check-hydra-logs" $ \dir -> do
+            withCardanoNodeDevnet (contramap FromCardanoNode tracer) dir $ \node -> do
+              let hydraSK = dir </> "hydra.sk"
+                  cardanoSK = dir </> "cardano.sk"
+                  cardanoLedgerGenesisFile = dir </> "genesis.json"
```

Comment:
This whole setup block could probably be done less verbose :(

Reply:
Yes. Wanna pair on this? Maybe tomorrow?

Reply:
I always wanna pair :)